dotcom-rendering/src/web/browser/coreVitals/init.ts

```diff
@@ -1,15 +1,31 @@
-import '../webpackPublicPath';
+import {
+	bypassCoreWebVitalsSampling,
+	getCookie,
+	initCoreWebVitals,
+} from '@guardian/libs';
 import { startup } from '@root/src/web/browser/startup';
-import { coreVitals } from './coreVitals';
-
-const init = async (): Promise<void> => {
-	// Sample 1 out of 100
-	const inSample = Math.floor(Math.random() * 100);
-	const inLocal = window.location.hostname === 'localhost';
-	if (inSample === 1 || inLocal) {
-		coreVitals();
-	}
+
+const coreVitals = (): Promise<void> => {
```

Comment:
```suggestion
const initCoreVitals = (): Promise<void> => {
```

Nitpick but most of these other init scripts are called 'init' - I think it's nice to maintain the convention that we're initialising something here?

Ideally, this code we use to initialised things like this would have a little more structure now that its grown some

Reply:
Sorry, I looked at the first two (`atomIframe` and `embedIframe`), and they didn’t follow this pattern. Happy to oblige